CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+-  Fixes [#4393](https://github.com/microsoft/BotFramework-WebChat/issues/4393). Assigns correct ARIA roles to components in web chat when a conversation status is pending, by [@beyackle2](https://github.com/beyackle2), in PR [#4420](https://github.com/microsoft/BotFramework-WebChat/pull/4420)
```

Comment:
```suggestion
### Fixed

-  Fixes [#4393](https://github.com/microsoft/BotFramework-WebChat/issues/4393). Assigns correct ARIA roles to components in web chat when a conversation status is pending, by [@beyackle2](https://github.com/beyackle2), in PR [#4420](https://github.com/microsoft/BotFramework-WebChat/pull/4420)
```

Reply:
I wasn't sure where this one belonged, since the section above "Fixed" is "Unreleased", and this change won't be released immediately.

Reply:
I think when TJ added his comment, this `### Fixed` was not in `CHANGELOG.md`.

A (few) recent PRs added the `### Fixed` section.